users/vieting/experiments/librispeech/librispeech_100_ctc/fairseq_finetuning/config_01_finetune.py

```diff
@@ -0,0 +1,185 @@
+"""
+Config for finetune experiments on LibriSpeech using wav2vec 2.0.
+"""
+import os.path
+
+from sisyphus import tk, gs
+import recipe.i6_core.datasets.librispeech as librispeech
+from recipe.i6_core.audio.encoding import BlissChangeEncodingJob
+from recipe.i6_core.tools.git import CloneGitRepositoryJob
+from recipe.i6_experiments.users.engler.fairseq.training import FairseqHydraConfig, FairseqHydraTrainingJob
+from recipe.i6_experiments.users.vieting.jobs.fairseq import CreateFairseqLabeledDataJob
+from recipe.i6_experiments.users.vieting.experiments.librispeech.librispeech_960_pretraining.wav2vec2.fairseq import SetupFairseqJob
+
+python_path = tk.Path("/work/asr3/vieting/hiwis/pletschko/miniconda3/envs/fairseq_python38/bin/python")
+
+
+def get_task(valid_percent=0.001, audio_format="ogg", output_prefix="datasets", corpus_names=["train-clean-100", "train-clean-360", "train-other-500"]):
```

Comment:
The default `valid_percent=0.001` seems to be very small. Is that common in fairseq? Also, this line seems too long. But if you use the helper function mentioned in the comment below, you can remove `corpus_names` I guess.